components/Board/SearchTitle.tsx

```diff
@@ -0,0 +1,124 @@
+"use client";
+import styles from "./SearchTitle.module.css";
+import Image from "next/image";
+import { List } from "@/app/apis/getArticle";
+import { formatDate } from "@/app/utils/formateDate";
+import heartIcon from "@/app/assets/images/ic_heart.svg";
+import profileIcon from "@/app/assets/images/ic_profile.png";
+import searchIcon from "@/app/assets/images/ic_search.png";
+import { useRouter } from "next/navigation";
+import { useState, useEffect } from "react";
+
+interface Props {
+  articles: List[];
+  keyword: string;
+  page: number;
+  pageSize: number;
+  orderBy: string;
```

Comment:
```suggestion
  orderBy: "recent" | "like";
```
위와 같이 타입을 좁혀주셔도 좋을 듯 합니다